docs/faq.md

```diff
@@ -35,12 +35,15 @@ Please read this page before asking a question in the Discord. Your help request
 **What OS should I use?**
 * This depends on your use case. Only you can answer this question.
 
-**My internal keyboard doesnt work, can I use an external keyboard to get into developer mode?**
+**My internal keyboard doesn't work, can I use an external keyboard to get into developer mode?**
 * No. The hotkeys will only work on the internal keyboard.
 
 **Should I use Windows 10 or 11?**
 * Windows 11 is required for TB4 users. If you don't have Thunderbolt 4, you may use both 10 or 11.
 
+**Will the audio driver still work when reinstalling windows?**
```

Comment:
"windows" should be capitalized. Also, explain that it'll only work if they're using the same machine and same storage device.

Reply:
Added and fixed